mico-core/src/main/java/io/github/ust/mico/core/model/MicoApplication.java

```diff
@@ -19,23 +19,23 @@
 
```

Comment:
Did you rearrange the code in the same PR as adding features?

Reply:
Yes. That is my current IntelliJ setting. I will change it.

Reply:
Only the imports were rearranged. Please check if the other changes are ok for you.